pokemon/src/main/java/gamarket/EncounterGUI.java

```diff
@@ -1,125 +1,1075 @@
+        // Background image for the pokemon wild encounter. 
+        filePath = "./pokemon/imgs/battle_background.png";
+        File file = new File(filePath);
+        String path3 = file.toURI().toString();
+        Image image3 = new Image(path3);
+        ImageView background = new ImageView(image3);
+        background.setFitHeight(300);
+        background.setFitWidth(300);
+        BackgroundSize bSize = new BackgroundSize(BackgroundSize.AUTO, 
+                                                    BackgroundSize.AUTO, false, false, true, true);
+        gp.setBackground(new Background(new BackgroundImage(image3, BackgroundRepeat.NO_REPEAT,
+                           BackgroundRepeat.NO_REPEAT, BackgroundPosition.CENTER, bSize)));
+ 
+        // Image for the fight button. 
+        filePath = "./pokemon/imgs/fight_button_image.png";
+        File test = new File(filePath);
+        String x = test.toURI().toString();
+        ImageView a = new ImageView(x);
+        a.setFitHeight(45);
+        a.setFitWidth(90);
+ 
+        // Image for the bag button. 
+        filePath = "./pokemon/imgs/bag_image.png";
+        File file2 = new File(filePath);
+        String f2 = file2.toURI().toString();
+        ImageView b = new ImageView(f2);
+        b.setFitHeight(45); 
+        b.setFitWidth(90);
+ 
+        // Image for the pokemon button. 
+        filePath = "./pokemon/imgs/pokeball.png";
+        File file3 = new File(filePath);
+        String f3 = file3.toURI().toString();
+        ImageView c = new ImageView(f3);
+        c.setFitHeight(45);
+        c.setFitWidth(80);
+ 
+        // Image for the run button. 
+        filePath = "./pokemon/imgs/running_icon.png";
+        File file4 = new File(filePath);
+        String f4 = file4.toURI().toString();
+        ImageView d = new ImageView(f4);
+        d.setFitHeight(65);
+        d.setFitWidth(120);
+ 
+        // Wild Pokemon image
+        enemyPoke = new ImageView();
+        enemyPoke.setImage(this.getImage(wildPokePath));
+ 
+        // Player Pokemon image
+        playerPoke = new ImageView();
+        playerPoke.setImage(this.getImage(playerPokePath));
+ 
+        // Image resizing for pokemons 
+        enemyPoke.setFitHeight(175);
+        enemyPoke.setFitWidth(175);
+        playerPoke.setFitHeight(175);
+        playerPoke.setFitWidth(175);
+ 
+        // BUttons
+        fight = new Button("Fight", a);
+        bag = new Button("Bag", b);
+        pokemon = new Button("Pokemon", c);
+        run = new Button("Run", d);
+ 
+        // Sync the Player's pokemon moves into the buttons.
+        move1 = new Button(); 
+        move1.setOnAction(e -> {this.move1();});
+        move2 = new Button(); 
+        move2.setOnAction(e -> {this.move2();});
+        move3 = new Button(); 
+        move3.setOnAction(e -> {this.move3();});
+        move4 = new Button(); 
+        move4.setOnAction(e -> {this.move4();});

```

Comment:
the move functions should be one function that takes an int moveNumber(int buttonNum)